assets/js/common/DismissableToast/DismissableToast.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { toast } from 'react-hot-toast';
+
+function DismissableToast({ text, toastInstance }) {
```

Comment:
Wouldn't be more logical to just get the `id` here -> `({ id, text})`?
Besides that, the `text` or the content could go as `children` so you have the chance to add other content to the toast, but it is true that right now we don't need it

Reply:
done with the id as props, for the children I will keep simple rn, we can modify later, the usage in the toast function call can be more cumbersome